### ISSUE TITLE
Make validator packages available through standard channels

### ISSUE THREAD

**Issue by bleonar5** [psych-ds/psychds-validator#24]
# TL;DR
For every version of the validator we develop, we should take the necessary steps to make them searchable/installable through the standard channels associated with that framework/language

- the node library should be available through npm
- the R library should be available through CRAN
- the python library should be available through Pip

All such public versions of the library should have protocols in place to coordinate their updates with upstream updates from their respective repos, and they should all be properly structured/documented according to the standards of each given platform.
